WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1835,16 +1814,6 @@ public void onPositiveClicked(@NonNull String instanceTag) {
 
     @Override
     public void onLinkClicked(@NonNull String instanceTag) {
```

Comment:
Better to remove that function completely, less lines in `EditPostActivity` = 🍰

Reply:
Right call!